supportedBackends/jes/src/main/scala/cromwell/backend/impl/jes/JesRuntimeAttributes.scala

```diff
@@ -110,11 +109,11 @@ object JesRuntimeAttributes {
     }
   }
 
-  private def validateZone(zoneValue: WdlValue): ErrorOr[Vector[String]] = {
+  private def validateZone(zoneValue: Option[WdlValue], defaultZones: NonEmptyList[String]): ErrorOr[Vector[String]] = {
```

Comment:
TOF (Thinking of Future): maybe we'll want to validate the zones available to a google project as well? but it seems outside the scope of this ticket.